variables.tf

```diff
@@ -68,3 +68,13 @@ variable "kms_alias" {
   description = "KMS key alias name for SQS"
   default     = ""
 }
+
+varialbe "redrive_arn" {
+  description = "AWS ARN for the SQS redirect queue"
+  default     = ""
+}
+
+varialbe "max_recieve_count" {
```

Comment:
varialbe --> variable
Shouldn't this be an integer instead of a string?

Also, neither of these two variables are actually being used.